### PR TITLE
Source LinkedInAds: Add dbt converter

### LINKED ISSUE #18: Create dbt converter for Airbyte Linkedin Ads source connector
The goal of this task is to ensure the Airbyte Linkedin Ads connector output can be used to run [Fivetran's dbt analytical model](https://hub.getdbt.com/fivetran/linkedin/latest/).

**Definition of Done:** 
* Create a pull request that adds new tables or views, enabling the execution of Fiventran's dbt with Airbyte output data.
* Update the documentation inside the connector folder to explain how to configure the user dbt package to run the analytical model.

**Resources:** 
* Check example of how it was done for [Zendesk Support](link) connector

### PR DESCRIPTION
**Pull request by ombhardwajj**
closes #18

### CODE REVIEW

File: connectors/source_linkedin_ads/models/fivetran_converter/account_history.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{% if target.type == "snowflake" %}
+
+    with tmp as (
```
Comment (marcosmarxm) [inline connectors/source_linkedin_ads/models/fivetran_converter/account_history.sql:3]: See you must create here what [this](https://github.com/fivetran/dbt_linkedin_source/blob/f953ac9f158990a525022da732237b6f2c14cae8/models/stg_linkedin_ads__account_history.sql#L27-L39) is consuming.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Comment by ombhardwajj** [comment]
I had a look at zendesk and amplitude @marcosmarxm .
 But I was not able to understand where issue is ,
I could only see a mistake that I did is  named  all my files with stg____
Like stg_linkedin_ads__creative_history.sql
Is this the only issue or am I missing something bigger? Please elaborate !🙇

**Comment by marcosmarxm** [comment]
> I had a look at zendesk and amplitude @marcosmarxm . But I was not able to understand where issue is , I could only see a mistake that I did is named all my files with stg____ Like stg_linkedin_ads__creative_history.sql Is this the only issue or am I missing something bigger? Please elaborate !🙇

I think you're in the path check my commit to `account_history`. Use this as example.

**Comment by ombhardwajj** [comment]
Hey @marcosmarxm 
I've made some changes , and I think files are looking good now!
Please LMK if any changes are needed!

**Comment by ombhardwajj** [comment]
Hey @marcosmarxm 
Any update...? 🙇

**Comment by ombhardwajj** [comment]
I've added readme , changed the schema according to recent changes in the postgres DB , and corrected the names in yaml files.
All the tests are running fine. 
But there is only one error , I tried a lot to fix it but it doesnt work out 🤔 :
`15:48:20  Completed with 1 error and 0 warnings:
15:48:20  
15:48:20    Database Error in model analytics_by_creative (models/fivetran_converter/analytics_by_creative.sql)
  column aca.costinusd does not exist
  LINE 15:             aca.costInUsd as cost_in_usd,
                       ^
  HINT:  Perhaps you meant to reference the column "aca.costInUsd".
  compiled Code at target/run/source_linkedin_ads/models/fivetran_converter/analytics_by_creative.sql
15:48:20  
15:48:20  Done. PASS=5 WARN=0 ERROR=1 SKIP=0 TOTAL=6`

**Comment by ombhardwajj** [comment]
Hey @marcosmarxm 
Did you have a look at the changes? 🙇

**Comment by marcosmarxm** [comment]
Oh I didn't push the changes lol!

**Review by marcosmarxm — APPROVED** [review]
Nice contribution @ombhardwajj

**Pull request merged**